sparkle/platform/output.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python3
+"""Sparkle class to organise output."""
+
+from __future__ import annotations
+
+from CLI.help import global_variables as gv
+# from sparkle.platform import generate_report_for_selection as sgfs
+from sparkle.platform import \
+    generate_report_for_configuration as sgrfch
+from sparkle.platform.settings_objects import Settings
+from sparkle.solver import Solver
+from sparkle.instance import InstanceSet
+from sparkle.configurator.configuration_scenario import ConfigurationScenario
+from sparkle.configurator.configurator import Configurator
+from sparkle.solver.validator import Validator
+from sparkle.types.objective import SparkleObjective
+
+from CLI.help.nicknames import resolve_object_name
+
+import os
+import json
+from pathlib import Path
+from runrunner.base import Status
+
+
+class ConfigurationPerformance:
+    """Class that stores performance results."""
+    def __init__(self: ConfigurationPerformance, configured_par10: float,
+                 default_par10: float) -> None:
+        """Initalize ConfigurationPerformance.
+
+        Args:
+            configured_par10: The PAR10 result of the configured solver
+            default_par10: The PAR10 result of the default solver
+        """
+        self.configured_PAR10 = configured_par10
+        self.default_PAR10 = default_par10
```

Comment:
PAR10 should be the same for the entire configuration scenario, so no difference between configured solver and default solver

Reply:
I'm confused, shouldn't the default and configured solver have different performance values? (I do get different values for configured and default)

Reply:
Yes see my other comment. You should name these something like configured_metrics, as PAR10 is a specific metric. I assumed you meant the penalty value here.